server/py/framework/db/sqldb/db.py

```diff
@@ -1662,6 +1662,7 @@ def _find_artifacts(
         query = session.query(ArtifactV2).with_entities(
             ArtifactV2.id,
             ArtifactV2.Tag.name,
+            ArtifactV2.Tag.id.label("tag_id"),  # Include tag_id for ordering
```

Comment:
move comment to have its own line (not inline)
also, explain why it is as label and not just as it is?